lib/pengine/common.c

```diff
@@ -237,49 +237,42 @@ static pcmk__cluster_option_t pe_opts[] = {
             PCMK__VALUE_ONLY_GREEN ", " PCMK__VALUE_PROGRESSIVE ", "
             PCMK__VALUE_CUSTOM,
         PCMK__VALUE_NONE, pcmk__validate_health_strategy,
-        "How cluster should react to node health attributes",
-        "Requires external entities to create node attributes (named with "
-            "the prefix \"#health\") with values \"" PCMK__VALUE_RED "\", "
-            "\"" PCMK__VALUE_YELLOW "\", or \"" PCMK__VALUE_GREEN "\"."
+        N_("How cluster should react to node health attributes"),
+        N_("Requires external entities to create node attributes (named with "
+            "the prefix \"#health\") with values \" PCMK__VALUE_RED \", "
```

Comment:
IIUC this way the macro won't get interpreted into "red" in the resulting "pacemaker-scheduled metadata"? And likewise in below? Hrm, I can tell how tricky string macros are for localization ...

Reply:
Sorry, I didn't think about that. It's really difficult, and I haven't thought of a better way than translating into strings.